packages/vscode-extension/tests/webviewUtils.ts

```diff
@@ -6,13 +9,25 @@ export async function waitUntilNotification(
     async function () {
       const wb = await browser.getWorkbench();
       const notifications = await wb.getNotifications();
-      const messages = await Promise.all(
-        notifications.map(async (n) => await n.getMessage()),
+
+      const notificationsAndMsgs = await Promise.all(
+        notifications.map(async (n) => {
+          const msg = await n.getMessage();
+          return { msg, notification: n };
+        }),
       );
 
-      return messages.includes(notification);
+      const found = notificationsAndMsgs.find(
+        (value) => value.msg === notification,
+      );
+      if (found) {
+        await found.notification.dismiss();
+        return true;
+      } else {
```

Comment:
We need this so we always close a notification when we've found it. Otherwise notifications keep piling up and if I connect to an instance and I disconnect, I would still be able to find the notification 'Connected to neo4j'